lib/puppet/type/cisco_interface.rb

```diff
@@ -728,100 +702,268 @@ def is_to_s(value)
   # private vlan attributes #
   ###########################
 
-  newproperty(:switchport_mode_private_vlan_host) do
-    desc 'Switchport private host mode of the interface.'
+  # ------------------------
+  newproperty(:pvlan_mapping, array_matching: :all) do
+    inputs =
+      'Valid inputs are a String containing a range of secondary vlans, '\
+      "example: '3-4,6'; or keyword 'default'"
+    desc 'Maps secondary VLANs to the VLAN interface of a primary VLAN. ' + inputs
+
+    validate do |value|
```

Comment:
Do we need to be concerned about an invalid manifest entry like this? `'5-2, 1'`